### PR TITLE
  [14.0] [MIG] purchase_order_general_discount: Migration to 14.0

### PR DESCRIPTION
**Pull request by zamberjo**
No code changes.

### CODE REVIEW

File: purchase_order_general_discount/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Order General Discount",
+    "summary": "General discount per purchase order",
+    "version": "14.0.1.0.0",
+    "category": "Purchases",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "author": "Tecnativa, Aures TIC, Odoo Community Association (OCA)",
```
Comment (HviorForgeFlow) [inline purchase_order_general_discount/__manifest__.py]: This field is intended to show original author of the module, neither migrations, nor minor changes contributors.

File: purchase_order_general_discount/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This module allows to set a general discount in a purchase order. This general
+discount is applied to each line order.
+
+You can also set a default general discount on suppliers and configure to wich
```
Comment (HviorForgeFlow) [inline purchase_order_general_discount/readme/DESCRIPTION.rst]: replace wich for which

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Code review. Minor comments

File: purchase_order_general_discount/views/res_config_view.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                        </div>
+                        <div class="content-group">
+                            <div class="mt16 row">
+                                <field
+                                    name="purchase_general_discount_field"
+                                    class="oe_inline"
+                                />
+                            </div>
+                        </div>
+                    </div>
```
Comment (HviorForgeFlow) [inline purchase_order_general_discount/views/res_config_view.xml]: The selection for this setting is missaligned with content:
image

Could you adapt what is done for Deposit Payment to align it? Thanks 

https://github.com/OCA/purchase-workflow/blob/14.0/purchase_deposit/views/res_config_settings_views.xml#L18-L29

File: purchase_order_general_discount/models/res_company.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    purchase_general_discount_field = fields.Selection(
+        selection="_get_purchase_discount_fields",
+        string="Purchase Discount Field",
+        default="discount",
+        help="Set the purchase line discount field in wich the "
```
Comment (HviorForgeFlow) [inline purchase_order_general_discount/models/res_company.py]: replace wich for which

File: purchase_order_general_discount/tests/test_purchase_order_general_discount.py [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    def _get_ctx_from_view(self, res):
+        order_xml = etree.XML(res["arch"])
+        order_line_path = "//field[@name='order_line']"
+        order_line_field = order_xml.xpath(order_line_path)[0]
+        return order_line_field.attrib.get("context", "{}")
+
+    def test_03_default_line_discount_value(self):
+        res = self.order.fields_view_get(
+            view_id=self.env.ref(
+                "purchase_order_general_discount." "purchase_order_form"
```
Comment (HviorForgeFlow) [inline purchase_order_general_discount/tests/test_purchase_order_general_discount.py]:
```suggestion
                "purchase_order_general_discount.purchase_order_form"
```

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Functional review LGTM :+1:

Some minor UI improvements

File: purchase_order_general_discount/models/res_company.py (hunk 4 above)
Comment (zamberjo) [reply purchase_order_general_discount/models/res_company.py]: I have replaced all occurrences of `wich`

**Review by HviorForgeFlow — APPROVED** [review]
Thanks for attend the comments quickly

CC @chienandalu @CarlosRoca13

**Review by chienandalu — APPROVED** [review]
LGTM :+1:

**Pull request merged**